src/JS/taskOperations.js

```diff
@@ -0,0 +1,92 @@
+let tasksList = [];
+
+const addOnLocalStorage = (task) => {
+  tasksList = JSON.parse(localStorage.getItem('tasks: '));
+  if (tasksList == null) tasksList = [];
+  tasksList.push(task);
+  localStorage.setItem('tasks: ', JSON.stringify(tasksList));
+};
+
+const addTaskToPage = (taskToBeDone) => {
+  const tasksContainerElt = document.querySelector('.tasks-container');
+  const taskDiv = document.createElement('div');
+  const taskContentDiv = document.createElement('div');
+  const taskValue = document.createElement('input');
+  const checkboxElt = document.createElement('input');
+  const trashIcon = document.createElement('i');
+  const dotsIcon = document.createElement('i');
+  taskDiv.setAttribute('class', 'task');
+  taskContentDiv.setAttribute('class', 'task-content');
+  taskValue.setAttribute('class', 'task-value');
+  taskValue.setAttribute('type', 'text');
+  checkboxElt.setAttribute('type', 'checkbox');
+  trashIcon.setAttribute('class', 'fa fa-trash trash-icon');
+  dotsIcon.setAttribute('class', 'fa fa-ellipsis-v dots-icon');
+  checkboxElt.checked = taskToBeDone.completed;
+  taskValue.value = taskToBeDone.description;
+  taskContentDiv.appendChild(checkboxElt);
+  taskContentDiv.appendChild(taskValue);
+  taskDiv.appendChild(taskContentDiv);
+  taskDiv.appendChild(trashIcon);
+  taskDiv.appendChild(dotsIcon);
+  tasksContainerElt.appendChild(taskDiv);
+};
+
+const loadTasksFromLS = () => {
+  tasksList = JSON.parse(localStorage.getItem('tasks: '));
+  if (tasksList !== null) {
+    tasksList.forEach((taskToBeDone) => {
+      addTaskToPage(taskToBeDone);
+    });
+  } else {
+    tasksList = [];
+  }
+};
+
```

Comment:
### Your MIT.md file
![todolist](https://user-images.githubusercontent.com/93039359/184036957-1f085557-2229-465f-a3cb-04d72e330016.jpg)
- [ ] Please add app type in your MIT.md file. You can add **WEBSITE**  or **WEBAPP**. This is the license file of your project so please take it seriously :thinking: